spring-cloud-starter-task-composedtaskrunner/src/main/java/org/springframework/cloud/task/app/composedtaskrunner/properties/ComposedTaskProperties.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.task.app.composedtaskrunner.properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties used to setup the ComposedTaskRunner.
+ *
+ * @author Glenn Renfro
+ */
+@ConfigurationProperties
+public class ComposedTaskProperties {
+
+	/**
+	 * The maximum amount of time in millis that a individual step can run before
+	 * the execution of the Composed task is failed.
+	 */
+	private int maxWaitTime = 30000;
+
+	/**
+	 * The amount of time in millis that the ComposedTaskStepExecutionListener
+	 * will wait between checks of the database to see if a task has completed.
+	 */
+	private int intervalTimeBetweenChecks = 500;
+
+	/**
+	 * The URI for the dataflow server that will receive task launch requests.
+	 */
+	private String dataFlowUri = "http://localhost:9393";
```

Comment:
I'm just surprised that we don't have something related to the SCDF Template that defaults this...